tasks/azure.py

```diff
@@ -0,0 +1,145 @@
+from ast import literal_eval
+from invoke import task
+from subprocess import check_output, run
+
+
+RESOURCE_GROUP = "faasm"
+LOCATION = "eastus"
+
+def _do_storage_account_get_key(name):
+    _az_cmd = [
+        "az",
+        "storage account keys list",
+        "--resource-group {}".format(RESOURCE_GROUP),
+        "--account-name {}".format(name),
+    ]
+
+    az_cmd = " ".join(_az_cmd)
+    print(az_cmd)
+
+    _out = check_output(az_cmd, shell=True).decode("utf-8")
+    out = literal_eval(_out)[0]["value"]
+    return out
+
+@task
+def storage_account_get_key(ctx, name):
+    """
+    Get key1 to access the storage account
+    """
+    key = _do_storage_account_get_key(name)
+    print(key)
+
+@task
+def storage_account_create(ctx, name, sku="Standard_LRS"):
```

Comment:
Can we put things like this `sku` and the vm type lower down into constants that live at the top? It's sort of config